pkg/process/checks/process.go

```diff
@@ -316,17 +393,29 @@ func fmtProcesses(
 		// Hide blacklisted args if the Scrubber is enabled
 		fp.Cmdline = cfg.Scrubber.ScrubProcessCommand(fp)
 
+		var ioStat *model.IOStat
+		if fp.Stats.IORateStat != nil {
```

Comment:
What's the difference btw this `IORateStat` and the `IOStat`?

Reply:
I had to abstract the rate (in /sec measurements) vs increasing values (that we convert to rates):
```
// IOCountersStat holds IO metrics for a process
type IOCountersStat struct {
	ReadCount  int64
	WriteCount int64
	ReadBytes  int64
	WriteBytes int64
}

// IOCountersRateStat holds IO metrics for a process represented as rates (/sec)
type IOCountersRateStat struct {
	ReadRate       float64
	WriteRate      float64
	ReadBytesRate  float64
	WriteBytesRate float64
}
```

Reply:
Took a minute for me to realized that `IOCountersRateStat` is used in Windows and the old `IOCountersStat` is used in others(am I right?). If it's the case I feel like explicitly calling them out as comments on the model file would make it easier to understand.